lib/remark-plugins/heading-ids/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Forked from https://github.com/imcuttle/remark-heading-id
+ * License MIT
+ * @author imcuttle
+ */
```

Comment:
I don't know what copyright to add here. Probably both the original author and zentered if they modified the code here.